src/main.rs

```diff
@@ -17,11 +17,7 @@ fn run() -> anyhow::Result<()> {
     env_logger::builder()
         .format_timestamp(None)
         .format_target(false)
-        .filter_level(match cli.verbosity {
-            0 => LevelFilter::Info,
-            1 => LevelFilter::Debug,
-            _ => LevelFilter::Trace,
-        })
+        .filter_level(cli.verbose.log_level_filter())
```

Comment:
With this change we don't get any errors at all. Now the loglevel is dropped to ERROR,
before it was at INFO.